clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -47,7 +47,12 @@
     CREATE_TOPICS(19, "CreateTopics"),
     DELETE_TOPICS(20, "DeleteTopics"),
     DELETE_RECORDS(21, "DeleteRecords"),
-    INIT_PRODUCER_ID(22, "InitProducerId");
+    INIT_PRODUCER_ID(22, "InitProducerId"),
+    ADD_PARTITIONS_TO_TXN(23, "AddPartitionsToTxn"),
+    ADD_OFFSETS_TO_TXN(24, "AddOffsetsToTxn"),
+    END_TXN(25, "EndTxn"),
+    WRITE_TXN_MARKER(26, "WriteTxnMarker"),
+    TXN_OFFSET_COMMIT(27, "TxnOffsetCommit");
```

Comment:
Is the GroupCoordinator -> FindCoordinator change and V2 of ListOffset happening later?